.github/workflows/backendbuild.yaml

```
@@ -0,0 +1,198 @@
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+
+jobs:
+  resolve-config:
+    runs-on: ubuntu-latest
+    outputs:
+      dockerfile_path: ${{ steps.pick_dockerfile.outputs.dockerfile_path }}
+      work_dir:        ${{ steps.pick_dockerfile.outputs.work_dir }}
+      tag:             ${{ steps.tag.outputs.tag }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)